splice_encoding/src/test/java/com/splicemachine/utils/MaskPatternLayoutTest.java

```diff
@@ -0,0 +1,63 @@
+package com.splicemachine.utils;
+
+import com.splicemachine.utils.logging.MaskPatternLayout;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.regex.Pattern;
+
+public class MaskPatternLayoutTest {
```

Comment:
Please include tests for incorrect patterns

Reply:
Incorrect pattern is handled outside the method `maskMessage`, so the only way to test it is to pass through log4j configuration, which is incompatible with other tests thus makes this ticket very complex. I've already manually tested it.